src/neo-vm/ExecutionEngine.cs

```diff
@@ -419,32 +421,276 @@ private bool ExecuteInstruction()
+                        break;
+                    }
+                case OpCode.STSFLD0:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 0)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD1:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 1)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD2:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 2)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD3:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 3)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD4:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 4)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD5:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 5)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD6:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, 6)) return false;
+                        break;
+                    }
+                case OpCode.STSFLD:
+                    {
+                        if (!ExecuteStoreToSlot(context.StaticFields, instruction.TokenU8)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC0:
+                    {
+                        if (!ExecuteLoadFromSlot(context.LocalVariables, 0)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC1:
+                    {
+                        if (!ExecuteLoadFromSlot(context.LocalVariables, 1)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC2:
+                    {
+                        if (!ExecuteLoadFromSlot(context.LocalVariables, 2)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC3:
+                    {
+                        if (!ExecuteLoadFromSlot(context.LocalVariables, 3)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC4:
+                    {
+                        if (!ExecuteLoadFromSlot(context.LocalVariables, 4)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC5:
+                    {
+                        if (!ExecuteLoadFromSlot(context.LocalVariables, 5)) return false;
+                        break;
+                    }
+                case OpCode.LDLOC6:
```

Comment:
@erikzhang @shargon can't we "simplify" this switch case part here by matching LDLOCX logic and subtracting from first opcode? It would remove several lines...
It could be less direct to understand, but less lines to maintain is usually better.

Reply:
Done.